packages/hint-no-broken-links/tests/tests.ts

```diff
@@ -108,75 +108,111 @@ const tests: Array<HintTest> = [
         reports: [{ message: `Broken link found (503 response).` }],
         serverConfig: {
-            '/': {content: generateHTMLPage('', bodyWithRelative503Links)},
-            '/503': {status: 503}
+            '/': { content: generateHTMLPage('', bodyWithRelative503Links) },
+            '/503': { status: 503 }
         }
     },
     {
         name: `This test should fail as it has a link with 404 href value`,
         reports: [{ message: `Broken link found (404 response).` }],
         serverConfig: {
-            '/': {content: generateHTMLPage('', bodyWithBrokenScriptTag)},
-            '/404': {status: 404}
+            '/': { content: generateHTMLPage('', bodyWithBrokenScriptTag) },
+            '/404': { status: 404 }
         }
     },
     {
         name: `This test should fail as it has a script with 404 src value`,
         reports: [{ message: `Broken link found (404 response).` }],
         serverConfig: {
-            '/': {content: generateHTMLPage('', bodyWithBrokenLinkTag)},
-            '/404': {status: 404}
+            '/': { content: generateHTMLPage('', bodyWithBrokenLinkTag) },
+            '/404': { status: 404 }
         }
     },
     {
         name: `This test should fail as it has an img with 404 src and srcset values`,
         reports: [
-            { message: `Broken link found (404 response).`},
-            { message: `Broken link found (404 response).`},
-            { message: `Broken link found (404 response).`}
+            { message: `Broken link found (404 response).` },
+            { message: `Broken link found (404 response).` },
+            { message: `Broken link found (404 response).` }
         ],
         serverConfig: {
-            '/': {content: generateHTMLPage('', bodyWithBrokenImageSrcSets)},
-            '/1.jpg': {status: 404},
+            '/': { content: generateHTMLPage('', bodyWithBrokenImageSrcSets) },
+            '/1.jpg': { status: 404 },
             '/2.jpg': '',
-            '/3.jpg': {status: 404},
-            '/4.jpg': {status: 404}
+            '/3.jpg': { status: 404 },
+            '/4.jpg': { status: 404 }
         }
     },
     {
         name: `This test should fail as it has a video tag broken poster and src`,
         reports: [
-            { message: `Broken link found (404 response).`},
-            { message: `Broken link found (404 response).`}
+            { message: `Broken link found (404 response).` },
+            { message: `Broken link found (404 response).` }
+        ],
+        serverConfig: {
+            '/': { content: generateHTMLPage('', bodyWithBrokenVideo) },
+            '/1.mp4': { status: 404 },
+            '/2.png': { status: 404 }
+        }
+    }
+];
+
+const loopTestsjsdom = [
+    {
+        name: `This test should fail as it has a loop`,
+        reports: [
+            { message: `Loop detected in url "http://localhost/1.mp4" using method GET` },
```

Comment:
To be consistent with the other error messages, maybe use something in the lines of: 

`'http://localhost/1.mp4' could not be fetched (redirect loop detected). `

Reply:
👍

Reply:
@sarvaje I'll take care of this before merging.